docs/elements.md

```diff
@@ -439,12 +439,13 @@ Given a list of statements, select a matching option for each entry from a drop-
-| `fixed-options-order` | boolean                                                    | False         | Whether or not to display the options in a fixed order; otherwise they are shuffled. See the details of `pl-option` below for more information on option ordering.                                                     |
+| `fixed-order`         | boolean                                                    | false         | Whether or not to display the statements in a fixed order; otherwise they are shuffled.                                                                                                                                |
+| `fixed-options-order` | boolean                                                    | false         | Whether or not to display the options in a fixed order; otherwise they are shuffled. See the details of `pl-option` below for more information on option ordering.                                                     |
 | `number-statements`   | integer                                                    | special       | The number of statements to display. Defaults to all statements.                                                                                                                                                       |
 | `number-options`      | integer                                                    | special       | The number of options to display. Defaults to all options. The `none-of-the-above` option does not count towards this number.                                                                                          |
 | `none-of-the-above`   | boolean                                                    | false         | Whether or not to add a "None of the above" to the end of the options.                                                                                                                                                 |
-| `blank`               | boolean                                                    | True          | Option to add blank dropdown entry as the default selection in each drop-down list.                                                                                                                                    |
+| `blank`               | boolean                                                    | true          | Option to add blank dropdown entry as the default selection in each dropdown list.                                                                                                                                     |
+| `allow-blank`         | boolean                                                    | false         | Whether or not an a blank submission is allowed. By default, empty dropdowns will not be graded (invalid format).                                                                                                      |
```

Comment:
Note that, unlike some other elements that allow a blank answer to be correct or to represent a different value, in this case a blank answer is always incorrect. I believe this should be clarified in the docs. This text is based on the same option in pl-multiple-choice (you will need to prettify the spacing).

```suggestion
| `allow-blank`         | boolean                                                    | false         | Whether or not a blank submission is allowed. If this is set to true, a statement that selects the blank entry will be marked as incorrect instead of invalid.  |
```

Reply:
Don't forget the prettify (`make format`).